samples/guestbook/main.go

```diff
@@ -123,22 +117,30 @@ var applicationSet = wire.NewSet(
 	newApplication,
 	appHealthChecks,
 	trace.AlwaysSample,
+	NewRouter,
+	wire.Bind((*http.Handler)(nil), (*mux.Router)(nil)),
 )
 
+func NewRouter(app *application) *mux.Router {
```

Comment:
nit: No need to export (it's in `package main`), so let's keep this unexported to match with surrounding code.

(Same comment applies to other NewRouter functions.)